modules/relax/models/mixins/hashMap.cfm

```diff
@@ -0,0 +1,25 @@
+<cfscript>
```

Comment:
You have to init it with an initial count.  If not, you get these issues.  Look here for example: https://github.com/ColdBox/coldbox-platform/blob/development/system/ioc/config/Binder.cfc#L123